packages/paste-website/src/components/empty-state/Error404.tsx

```diff
@@ -14,26 +13,20 @@ interface Error404Props {
 
 const Error404 = ({componentList, layoutList, primitiveList}: Error404Props): React.ReactNode => {
   const router = useRouter();
-  const pathParts = router.pathname.split('/');
+  const pathParts = router.asPath.split('/');
   const pageName = pathParts[pathParts.length - 1];
   const packageName = `@twilio-paste/${pageName}`;
   const packageObj = [...componentList, ...layoutList, ...primitiveList].find(({name}) => name === packageName);
-
   if (packageObj != null) {
     const isInDevelopment = packageObj.status !== PackageStatus.BACKLOG;
 
-    if (router.pathname.includes(SidebarCategoryRoutes.COMPONENTS)) {
-      if (isInDevelopment) {
+    if (isInDevelopment) {
+      if (router.asPath.includes(SidebarCategoryRoutes.COMPONENTS)) {
         return <InDevelopment type="component" name={pageName} />;
       }
-      return <NotBuilt type="component" name={pageName} />;
-    }
-
-    if (router.pathname.includes(SidebarCategoryRoutes.PRIMITIVES)) {
-      if (isInDevelopment) {
+      if (router.asPath.includes(SidebarCategoryRoutes.PRIMITIVES)) {
         return <InDevelopment type="primitive" name={pageName} />;
       }
-      return <NotBuilt type="primitive" name={pageName} />;
```

Comment:
We no longer have placement packages for stuff we want to build. So this condition never gets met and we would never show this "it's on the backlog page"